src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -0,0 +1,228 @@
+﻿using Dynamo.Core;
+using Dynamo.Models;
+using Dynamo.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace Dynamo.Core
+{
+    internal class CrashReportArgs : EventArgs
+    {
+        public bool SendLogFile = true;
+        public bool SendSettingsFile = true;
+        public bool SendDynFile = true;
+        public bool SendRecordedCommands = true;
```

Comment:
Are these flags going to be controlled somewhere (in the CER UI)?

Reply:
These flags are not exposed/controlled by the CER UI. 
I added these flags expecting that I will expose the CrashReportTool as a public API for host integrators.
Some might want less to be collected ....